fitPanel/controller/SimpleFitPanel.controller.js

```diff
@@ -8,29 +8,14 @@ sap.ui.define([
 ], function (GuiPanelController, JSONModel, ColorPickerPopover, MessageBox, MessageToast, Button) {
    
    "use strict";
+<<<<<<< HEAD
 
-//    return sap.ui.core.Control.extend("Button", { // call the new Control type "my.ColorBox" and let it inherit from sap.ui.core.Control
-
-//             // the control API:
-//             metadata : {
-//                properties : {           // setter and getter are created behind the scenes, incl. data binding and type validation
-//                   "color" : {type: "sap.ui.core.CSSColor", defaultValue: "#fff"} // you can give a default value and more
-//                }
-//             },
-
-//             // the part creating the HTML:
-//             renderer : function(oRm, oControl) { // static function, so use the given "oControl" instance instead of "this" in the renderer function
-//                oRm.write("<div"); 
-//                oRm.writeControlData(oControl);  // writes the Control ID and enables event handling - important!
-//                oRm.addStyle("background-color", oControl.getColor());  // write the color property; UI5 has validated it to be a valid CSS color
-//                oRm.writeStyles();
-//                oRm.addClass("myColorBox");      // add a CSS class for styles common to all control instances
-//                oRm.writeClasses();              // this call writes the above class plus enables support for Square.addStyleClass(...)
-//                oRm.write(">"); 
-//                oRm.write("</div>"); // no text content to render; close the tag
-//             },
-//          });
+=======
+<<<<<<< HEAD
+=======
 
+>>>>>>> 886ec6c... Taking the function from ComboBox or by User Input
+>>>>>>> Take function parameter from ComboBox or from User input/Remove custom css
    return GuiPanelController.extend("localapp.controller.SimpleFitPanel",{
```

Comment:
@ilianabe See, there is marks from `git rebase` commands.
You need to remove them, add new commit and push changes.